generators/form/templates/formComplex.js.ejs

```diff
@@ -87,10 +87,10 @@ const formConfig = {
     usaPhone,
   },
   chapters: {
-    chapterApplicantInfo: {
+    chapterApplicantInformation: {
```

Comment:
@cehsu I saw you mention in your review of BAH's PR that you didn't like the naming convention of chapters starting with `chapter...`. That's something that came from this form generator. I added that convention because I was trying to add a little bit of clarity to the keys of the form config. Especially when I was brand new to this, it was easy to get lost when looking at a massive form config object. Having something clearly labeled as a `chapter` serves as a landmark to help you figure out where you are in the object.